frontend/common/access/RolesWizard/steps/SelectRolesStep.tsx

```diff
@@ -0,0 +1,38 @@
+import { useFormContext } from 'react-hook-form';
+import { ISelected, ITableColumn, IToolbarFilter, IView } from '../../../../../framework';
+import { useEffect } from 'react';
+import { PageMultiSelectList } from '../../../../../framework/PageTable/PageMultiSelectList';
+import { Title } from '@patternfly/react-core';
+import { useTranslation } from 'react-i18next';
+import styled from 'styled-components';
+
+interface SelectRolesStepProps<T extends object> {
+  view: IView & ISelected<T> & { itemCount?: number; pageItems: T[] | undefined };
+  tableColumns: ITableColumn<T>[];
+  toolbarFilters: IToolbarFilter[];
+}
+
+const StyledTitle = styled(Title)`
+  margin-bottom: 4vh;
```

Comment:
that's a curious margin size. is that copied from somewhere else?

Reply:
yes, sorry about that! Fixed 👍